normative-conventions.md

```diff
@@ -37,3 +37,11 @@ For the purposes of this guideline `-0` is considered to be an integral number (
 Some APIs intentionally round non-integral inputs, for example as an attempt to provide a best-effort behavior, rather than because the API fundamentally only makes sense with integers. This guideline does not apply to those cases.
 
 NB: This convention is new as of 2024, and most earlier parts of the language do not follow it.
+
+## Reject primitives in iterable-taking positions
+
+Any time an iterable or async-iterable value (a value that has a `Symbol.iterator` or `Symbol.asyncIterator` method) is expected, primitives should be treated as if they were not iterable. Usually, this will mean throwing a `TypeError`. Primitive wrapper Objects such as String Objects, however, should be treated like any other Object.
```

Comment:
I don't know how to interpret "Primitive wrapper Objects such as String Objects, however, should be treated like any other Object.". The only way the language can get at `String.prototype[Symbol.iterator]` is via a String object wrapper. I guess you mean something like primitive wrapper Objects that are explicitly created by user code instead of implicitly? I don't know how to nail down "implicitly" yet.

Reply:
```suggestion
Any time an iterable or async-iterable value (a value that has a `Symbol.iterator` or `Symbol.asyncIterator` method) is expected, primitives should be treated as if they were not iterable. Usually, this will mean throwing a `TypeError`. If the user passes a primitive wrapper Object such as a String Object, however, should be treated like any other Object.
```

maybe?

Reply:
Yeah, that reads better to me.